contract/src/liquid.mligo

```diff
@@ -107,11 +107,11 @@ let validate_redemption ( amount_to_redeem, storage : nat * liquid_storage ) : n
 
 (* Calculates the amount of evXTZ to be minted from the required XTZ that is being deposited *)
 let get_amount_to_mint ( a, r : nat * exchange_rate) : nat =
-    (a / r.xtz) * r.evxtz
+    a * r.evxtz / r.xtz
```

Comment:
Truncating integer division is happening if division comes first. :(

Reply:
Good catch!